index.js

```diff
@@ -1,58 +1,48 @@
 /**
  * PCM volume controller.
- * Same as GainNode in web-audio-api, but for streams.
+ * Same as GainNode in web-audio-api as a generic function.
+ * stream in stream.js, pull-stream in pull.js, element in element.js
  *
  * @module audio-gain
  */
-
-var inherits = require('inherits');
-var Through = require('audio-through');
-var extend = require('xtend/mutable');
+var noop = function () {};
 var util = require('audio-buffer-utils');
 
 /**
- * Create pcm stream volume controller.
+ * Create pcm volume controller.
  *
- * @constructor
+ * @function
  */
-function Gain (volume) {
-	if (!(this instanceof Gain)) return new Gain(volume);
-
-	if (typeof volume === 'number') {
-		volume = {volume: volume};
+function gain(options) {
+	// gain()
+	if (typeof options === 'undefined') options = 1;
+
+	// gain(Number)
+	// gain({volume: Number})
+	if (typeof options === 'number' || typeof options.volume === 'number') {
+		var volume = Math.tan(options.volume || options);
```

Comment:
I would suggest gating this as a `option.mode` property.
There might be `'mult'`, `'tan'` and `'db'`, for example.
In `db` case it is possible to set values by changing decibels, like `volume = -20` means decreas by 20db.
But it's a subject of a new PR.